backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc

```diff
@@ -38,11 +38,16 @@ spec:
       - openshift
       - aws
       - csi <3>
+      defaultSnapshotMoveData: true
+      defaultVolumesToFSBackup: <4>
+      featureFlags:
+      - EnableCSI
 # ...
 ----
 <1> The flag to enable the node agent.
 <2> The type of uploader. The possible values are `restic` or `kopia`. The built-in Data Mover uses Kopia as the default uploader mechanism regardless of the value of the `uploaderType` field.
 <3> The CSI plugin included in the list of default plugins.
+<4> Set to `true` if you use Data Mover only for volumes that opt out of `fs-backup`. Set to `false` if you use Data Mover by default for volumes.
```

Comment:
DPA flags are introduced in 1.3.1 here you can keep the OADP 1.3.1 and later string but for backup spec it's not needed